src/stats.c

```diff
@@ -6,7 +6,7 @@ void r_stats_init(struct RaucStats *stats)
 
 	memset(stats, 0, sizeof(*stats));
 
-	stats->min = G_MAXDOUBLE;
+	stats->max = G_MAXDOUBLE;
 	stats->min = G_MINDOUBLE;
```

Comment:
This must be the other way around. If `max` is already `G_MAXDOUBLE`, it can't change anymore.

Reply:
Indeed, now both are broken. Will fix.